Tasks/google-play-release/GooglePlay.ts

```diff
@@ -107,13 +105,20 @@ async function run() {
 
         let requireTrackUpdate = false;
         if (shouldUploadApks) {
-            tl.debug(`Uploading ${apkFileList.length} APK(s).`);
+            tl.debug(`Uploading ${Object.keys(apkToObbMapping).length} APK(s).`);
             requireTrackUpdate = true;
 
-            for (const apkFile of apkFileList) {
+            for (const apkFile of Object.keys(apkToObbMapping)) {
                 tl.debug(`Uploading APK ${apkFile}`);
+                console.log(apkFile);
                 const apk: googleutil.Apk = await googleutil.addApk(edits, packageName, apkFile);
                 tl.debug(`Uploaded ${apkFile} with the version code ${apk.versionCode}`);
+                if (apkToObbMapping[apkFile] !== null) {
+                    const obb: googleutil.ObbResponse = await googleutil.addObb(edits, packageName, apkToObbMapping[apkFile], await getVersionCodeForApk(apkFile), 'main');
```

Comment:
Is `getVersionCodeForApk` necessary?  I thought `apk.versionCode` should work.  Is it not what you need?